tools/docker/env/Dockerfile

```diff
@@ -112,6 +112,19 @@ RUN chmod 0777 /syzkaller
 ENV SOURCEDIR_FUCHSIA /syzkaller/fuchsia
 ENV SOURCEDIR_NETBSD /syzkaller/netbsd
 
+# Latest Rust compiler.
+ENV RUSTUP_HOME=/usr/local/rustup
+ENV CARGO_HOME=/usr/local/cargo
+ENV PATH=/usr/local/cargo/bin:$PATH
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain 1.86.0
```

Comment:
https://wiki.debian.org/Rust - rustup also available in repos since Debian 13. Might be less scary than 'curl | sh', although it appears generally safe given rustup.rs as the source of the installer.

Reply:
We don't have Debian 13 as far as I see yet -> https://www.debian.org/releases/trixie/ .
Marco, what do you propose to do instead?

Reply:
No action if we only have Debian 13.